src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,239 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String createRequest = "INSERT INTO cars (manufacturer_id, model) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createStatement
+                        = connection.prepareStatement(createRequest,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setLong(1, car.getManufacturer().getId());
+            createStatement.setString(2, car.getModel());
+            createStatement.executeUpdate();
+            ResultSet resultSet = createStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't add new car: "
+                    + car.toString() + " to cars DB. ", e);
```

Comment:
```suggestion
                    + car + " to cars DB. ", e);
```

we don't need to call `toString()` explicitly